src/components/common/playlist/PlaylistDetails.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Image, PixelRatio, StyleSheet, Text, View } from "react-native";
+import PropTypes from "prop-types";
+import MoreIcon from "../icons/MoreIcon";
+import PlayIcon from "../icons/PlayIcon";
+import { handleStartPlaylist } from "../../../utils/MusicPlayerUtil";
+import { COLOR_PALETTE } from "../../../utils/constants";
+
+const PlaylistDetails = ({ playlist }) => {
```

Comment:
Let's move `common/playlist/` archive to the component level, so we have `component/playlist/` hope that makes sense.